codegen/bin/codegen-ios

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var path = require('path');
+var mkdirp = require('mkdirp');
+var semver = require('semver');
+var optimist = require('optimist');
+
+// var generator = require(loopback-sdk-ios-codegen);
+var generator = require('../lib/objc-codegen');
+
+var argv = optimist
+  .usage('Generate iOS Objective-C client code for your LoopBack application.' +
+    '\nUsage:' +
+    '\n    $0 [options] path-to-app ios-client-dir-path' +
+    '\n' +
+    '\nExample:' +
+    '\n    $0 -p XX server/app.js client/ios/gen-src')
+  .describe('p', 'The prefix for generated iOS object names.')
+  .default('p', '')
+  .alias({ p: 'model-name-prefix' })
+  .demand(2)
+  .argv;
+
+var appFile = path.resolve(argv._[0]);
+var clientDir = path.resolve(argv._[1]);
+var modelPrefix = argv['model-name-prefix'];
+
+console.error('Loading LoopBack app %j', appFile);
+var app = require(appFile);
+// assertLoopBackVersion(appFile);
```

Comment:
Turning on the asserting, I'm getting the following error:
```
Loading LoopBack app "/Users/hideya/Desktop/loopback-sdk-ios/CodeGen/test-env/server/server"
module.js:338
throw err;
^
Error: Cannot find module 'loopback'
at Function.Module._resolveFilename (module.js:336:15)
at Function.Module._load (module.js:278:25)
at assertLoopBackVersion (/Users/hideya/Desktop/loopback-sdk-ios/CodeGen/bin/codegen-ios:63:25)
...
```
I'm not sure why it cannot find loopback since it is installed (can be found under `node_modules`, and the test app runs fine...  @bajtos would you have any idea?